supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -323,13 +323,13 @@ There is no functional difference between the first server that was installed an
 *See also*:
 
 [[ingress]]
-==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+==== image:images/yes.png[yes] ingress (noun, adjective)
+*Description*: In Red Hat OpenShift, _Ingress_  is a Kubernetes API object that developers can use to expose services within the cluster through an HTTP(S) aware load balancer and a proxy layer using a public DNS entry. The `Ingress` resource defines the cluster-wide configuration for ingress traffic, and provides the ability to specify TLS options, a certificate, or a public CNAME that the OpenShift `IngressController` resource can accept for HTTP(S) traffic. Additionally, _ingress_ can also be used to describe the incoming direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.
```

Comment:
Thanks, @skrthomas , for adding this item. I've queried these trms in the past as they are highlighted illogically sometimes in customer documentation. 

Just wondering about "HTTP(S) aware load balance": I understand with the paranethesis, appending "aware" with a hyphen, which would be common, might be difficult, so how about: "an HTTP(S) load balancer" to drop the potential hyphenated construct?

Expand CNAME?

"IngressController object" more precise? I see the object reference commonly adopted in OCP docs. 

The "See also" subheading needs to be escaped onto the next line.

Reply:
@dfitzmau I had the same question about whether or not to hyphenate "HTTP(S) aware load balance". If removing "aware" works, that would solve the issue. @skrthomas wdyt?

Reply:
I think dropping the "aware" is totally fine. Good catch :)  It was a holdover from the original definition that I didn't change but could definitely be changed without altering the meaning.

Reply:
>Expand CNAME?
>"IngressController object" more precise? I see the object reference commonly adopted in OCP docs.

I do not think CNAME needs expanding.
Yea, I think `Ingress` resource and `IngressController` object is precise. thanks for pointing this out!